src/CoreWCF.Templates/tests/BasicTests.cs

```diff
@@ -66,10 +79,20 @@ public TestVariation Framework(string framework)
             return this;
         }
 
-        public static implicit operator object[](TestVariation testVariation) => new object[] { testVariation };
+        public void Deserialize(IXunitSerializationInfo info)
+        {
+
+        }
+
+        public void Serialize(IXunitSerializationInfo info)
+        {
+            info.AddValue(nameof(Arguments), JsonConvert.SerializeObject(Arguments));
```

Comment:
JSON.NET is being semi-deprecated with the introduction of [System.Text.Json](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/overview). If it doesn't introduce too much extra code, I would prefer to use that. It has first class support for AOT and in the far future we probably want to support AOT scenarios and this would be one more thing needing to be fixed if we stick with JSON.NET.

Reply:
Done, I drop Json.NET usage in favor of STJ.